data-analysis/graphs.py

```diff
@@ -0,0 +1,183 @@
+import matplotlib.colors as clr
+import matplotlib.cm as cm
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+import squarify 
+
+def init_setup():
+
+    sns.set(rc={'figure.figsize':(15,8)})
+    sns.set(style="white", context="talk")
+   
+    
+def genre_pie(genre_counts, genre_list, name):
+
+    fig, ax = plt.subplots(subplot_kw=dict(aspect="equal"))
+    wedges, texts = ax.pie(genre_counts)
+
+    #Compute percentages
+    pct = 100*genre_counts/len(genre_list)
+
+    ax.legend(labels=['%s  %1.2f %%' % (l, s) for l, s in zip(list(pct.keys()), list(pct))], loc="center left",bbox_to_anchor=(1, 0, 0.5, 1))
+    plt.title(name)
+    plt.show()
+    
+    
+def genre_bar(genre_counts, genre_list, name): 
+    
+    total = float(len(genre_list))
+    ax = sns.barplot(x=genre_counts, y=genre_counts.keys(), orient='h', palette='PuBu_r')
+
+    #Compute and locate percentages
+    for p in ax.patches:
+        loc = ax.patches[0].get_width()
+        width = p.get_width()
+        ax.text(loc+ 20, p.get_y()+p.get_height()/1.5, '  '+'{:1.2f}'.format(100*width/total) + '%', ha='left') 
```

Comment:
when you use .format, you don't need to use + because you can put the whole string in a single set of quotes:
`' {:1.2f}%'.format(100*width/total)`